src/terra-dev-site/doc/UpgradeGuides.c/v4.0.0.v.doc.md

```diff
@@ -1,5 +1,5 @@
 # v4.0.0 Upgrade Guide
-This document will provide information on upgrading from Terra Toolkit `^3.0.0` to `4.0.0`.
+This document will provide information on upgrading from Terra Dev Site `^3.0.0` to `4.0.0`.
 
 ## Dependency Changes
```

Comment:
There's a typo below (saw in the screenshot)

Reference [terra-base's vt upgrade guide] => Reference [terra-base's v5 upgrade guide]